server.go

```diff
@@ -82,10 +85,11 @@ func (s *oidcServer) authorization(w http.ResponseWriter, req *http.Request) {
 
 	// stash in session, so we can pull it out in the login handler without
 	// threading it through the user code. make sure to clear it though!
-	sess := sessionFromContext(req.Context())
+	sess, _ := s.sessmgr.Get(req.Context())
```

Comment:
This is more about cookiesession but kinda weird to ignore the existed bool everywhere. Wonder if that should be split up into like `GetFromContext`? Or I guess why return that bool at all since `Get` panics.

Reply:
The bool indicates if the session already existed, or if it was newly created. Either way, we get a usable session object. In our case we don't care that anything was loaded, we just set what we need. 

I kept it there for a case where it's like "oh yeah, we're starting something new so I wanna do X". But now I think about it, maybe there's not a case where external indications of that are useful? If you wanna do new object initialisation, you can implement https://github.com/lstoll/cookiesession/blob/1f1508aa7ff654381e679392e7373b580e103120/session.go#L35-L39 . Otherwise if you wanna track that it existed, add your own bool to the object? I might just drop the bool.

Reply:
I'm gonna land this now, but I'm gonna go back and drop the bool from `cookisessions`